.github/workflows/moodle-ci.yml

```diff
@@ -0,0 +1,161 @@
+# Jobs: 
+# - "test"
+#
+# Runs tests and code analysis ( like PHPUnit, Behat, code checker, etc ) 
+# against a Moodle plugin in CI.
+# - See https://moodlehq.github.io/moodle-plugin-ci/.
+# 
+# The current matrix we are using is:
+# - ubuntu-latest
+# - PHP8.0, PHP8.1, PHP8.2, PHP8.3
+# - Moodle from 4.2 to latest(4.4)
+
+name: Moodle Plugin CI
+
+# Run this workflow every time a new commit pushed to the repository or PR
+# is created or on demand. It also executes every morning at 6:30 so we're 
+# the first to know if tests are failing due to an update on PHP or Moodle.
+on:
+  schedule:
+    - cron: '30 6 * * *'
+  push:
+  workflow_dispatch:
+
+jobs:
+  test:
+    # We will use ubuntu-latest as a virtual environment to use.
+    runs-on: ubuntu-latest
```

Comment:
Lo comento pero no es importante ni motivo para parar el PR.

El comentario dice lo mismo que la línea que comenta ("we will use ubuntu-latest as ... environment" == "runs-on: ubuntu-latest"), los que defienden no comentar suelen basarse en casos como este o:

```
// Obtiene el código postal
fn get_postal_code() {
```

Y los que defienden el comentar y documentar apoyan el dar información contextual que el código no puede dar, por ejemplo, por qué usar ubuntu-latest, o por qué una versión x, etc.

Dejo un [podcast de Carlos Blé sobre esto interesante](https://savvily.es/podcasts/ni-cero-ni-uno/episodios/e12).